clerk-frontend/public/index.html

```diff
@@ -1,40 +1,136 @@
+    <div>
+      <header class="header">
+        <div class="wrapper">
+          <div id="top" class="top">
+            <a class="button-basic" id="button-menu">
+    				<span aria-hidden="true"></span>
+    				Menu
+    			</a>
+            <h1 class="logo">
+    				<a title="Gemeente Haarlem"> Gemeente <span>Haarlem</span></a>
+    			</h1>
+            <ul class="skip-links">
+              <li><a>Naar de inhoud</a></li>
+              <li><a>Naar hoofdnavigatie</a></li>
+            </ul>
+            <a class="button-basic" id="button-search">
+    				<span aria-hidden="true"></span>
+    				Zoek
+    			</a>
+          </div>
+          <ul class="top-links">
+            <li class="top-link"><a title="Toeristische informatie" rel="external">Toeristische informatie<span class="screen-reader-text"> (externe website)</span><span aria-hidden="true"></span></a></li>
+          </ul>
+          <form id="tx-solr-search-form-pi-search" action="https://www.haarlem.nl/zoeken" class="search" method="get" accept-charset="utf-8">
+            <fieldset>
+              <legend class="screen-reader-text">Zoeken</legend>
+              <div>
+                <span aria-hidden="true"></span>
+                <label for="inputText" class="screen-reader-text">Zoeken</label>
+                <input type="hidden" name="id" value="70">
+                <input type="hidden" name="L" value="0" id="search">
+                <input type="search" id="inputText" class="tx-solr-q" name="q" value="">
+              </div>
+              <input type="submit" value="Zoek" class="button">
+            </fieldset>
+          </form>
+        </div>
+      </header>
+      <nav id="navigation">
+        <h2 class="visuallyhidden">Navigatie</h2>
+        <ul class="wrapper">
+          <li class="tab-citizens is-selected"><a title="Inwoners en ondernemers">Inwoners en ondernemers</a>
+            <ul class="sub">
+              <li class="is-selected"><a title="Paspoort, rijbewijs en uittreksel">Paspoort, rijbewijs en uittreksel</a>
+              </li>
+              <li><a title="Parkeren, verkeer en vervoer">Parkeren, verkeer en vervoer</a>
+              </li>
+              <li><a title="Afval en milieu">Afval en milieu</a>
+              </li>
+              <li><a title="Geboorte, huwelijk en overlijden">Geboorte, huwelijk en overlijden</a>
+              </li>
+              <li class="last more"><a title="Meer onderwerpen...">Meer onderwerpen...<span aria-hidden="true"></span></a>
+              </li>
+            </ul>
+          </li>
+          <li class="tab-citydistricts"><a title="Stadsdelen">Stadsdelen</a>
+          </li>
+          <li class="tab-governance"><a title="Bestuur">Bestuur</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="content">
+        <div class="wrapper">
+          <div class="breadcrumbs">
+            <ol class="breadcrumb">
+              <li><a>Home<span aria-hidden="true"></span></a></li>
```

Comment:
These breadcrumbs look a little off to me, because there's no spacing between them. I don't feel this is a showstopper, but it would probably look nicer. (I imagine the actual Haarlem site doesn't have this.)